common/src/main/java/com/wynntils/features/combat/RangeVisualizerFeature.java

```diff
@@ -80,25 +91,22 @@ private CustomColor getCircleColor(Player player) {
 
         if (isSelf) { // Do not render if the item is not for the player's class
             if (gearInfo.requirements().classType().isEmpty()) return null;
+
             ClassType classType = gearInfo.requirements().classType().get();
             if (classType != Models.Character.getClassType()) return null;
         }
 
-        // Major IDs that we can visualize:
-        // Taunt (12 blocks)
-        // Saviour's Sacrifice (8 blocks)?
-        // Heart of the Pack (8 blocks)?
-        // Guardian (8 blocks)?
-        // Marked with a ? needs additional confirmation
-
-        if (gearInfo.fixedStats().majorIds().isEmpty()) return null;
-
-        return switch (gearInfo.fixedStats().majorIds().get(0).name()) {
-            case "HERO" -> CommonColors.WHITE;
-            case "ALTRUISM" -> CommonColors.PINK;
-            case "GUARDIAN" -> CommonColors.RED;
-            default -> null;
-        };
+        return gearInfo.fixedStats().majorIds().stream()
+                .map(majorId -> switch (majorId.name()) {
+                    case "TAUNT" -> Pair.of(CommonColors.ORANGE, 12);
```

Comment:
I don't think we should have taunt, there is actually no item at all that we can hold that has the major ID taunt

Reply:
Oh, I see, taunt is only on armor. 

But should we not then check armor as well..?

Reply:
Taunt is also on an accessory, which is impossible to check, so that's why I decided to remove it, then it would at least be consistent

Reply:
(Taunt is available on "Signal Flare", legendary boots, and "Strobelight", a fabled necklace (with Warrior req))

Reply:
(Taunt is available on "Signal Flare", legendary boots, and "Strobelight", a fabled necklace (with Warrior req))

Reply:
Hm oh actually... the new event chestplate has the hero id as well...

Reply:
But we can check accessories for ourselves, at least. Just because we can't cover all cases does not mean we need to exclude cases we *can* cover, right?

Reply:
I added check for armor on all players, and accessories for ourselves. We can unfortunately never get full detection on other players. :(